cmd/helm/search.go

```diff
@@ -63,7 +63,7 @@ func newSearchCmd(out io.Writer) *cobra.Command {
 	}
 
 	f := cmd.Flags()
-	f.BoolVarP(&sc.regexp, "regexp", "r", false, "use regular expressions for searching")
+	f.BoolVarP(&sc.regexp, "regexp", "r", false, "use regular expressions for searching in the concatenation of multiple fields separated by '\\v' (vertical tab)")
```

Comment:
Hey! Would you mind adding this to the description of `helm search` instead, on line 33?

Something like

```
Search reads through all of the repositories configured on the system and looks for matches.
 
Repositories are managed with 'helm repo' commands.

To look for charts with a particular name (such as stable/mysql), try searching using vertical tabs \v:

helm search --regexp '\vstable/mysql\v'
```

Let me know if that won't work for you. Thanks!

Reply:
Definitely a better way to document it!
I used your example and just added line breaks to fit the 80 cols convention.